src/settings.js

```diff
@@ -1395,6 +1395,10 @@ var FETCH_DEBUG = 0;
 // If nonzero, enables emscripten_fetch API.
 var FETCH = 0;
 
+// Whether to use an asm.js fetch worker when using FETCH. Note that this is
+// only relevant for fastcomp, where we support asm.js.
+var USE_FETCH_WORKER = 1;
```

Comment:
Sorry to nit pick, but I think it would be good to mention that the fetch worker is only needed/used in synchronous fetched in the main thread.
Currently it sounds like this option will proxy all FETCH calls through a worker.

Reply:
It does, actually: in asm.js when a fetch worker is used, it does all sync operations through it, even if it it could do it directly (which it can off the main thread).

From reading the code, I think that's because it's written to assume the worker handles all IndexedDB stuff, so the idea is to let the worker do all sync fetch stuff. But I'm not sure.

Reply:
Oh OK, I was mistaken.